source/operations/install-deploy-manage/deploy-minio-tenant.rst

```diff
@@ -110,442 +110,528 @@ volume and a supporting
+           To specify a different version, use the :mc:`~kubectl minio tenant create --image` flag.
+
+           See the `MinIO Quay <https://quay.io/repository/minio/minio>`__ or the `MinIO DockerHub <https://hub.docker.com/r/minio/minio/tags>`__ repositories for a list of valid tags.
+
+         - :mc:`~kubectl minio tenant create --image-pull-secret`
+           
+           If required, specify the secret to use when pulling the ``minio`` image.
+
+           Use :mc:`~kubectl minio tenant create --image-pull-secret` to specify the secret.
+
+         - :mc:`~kubectl minio tenant create --kes-config`
+           
+           Configure a :minio-git:`Key Encrption Service (KES) <kes>`
+
+           Use the :mc:`~kubectl minio tenant create --kes-config` flag to specify the name of the secret to use for KES Key Management Service (KMS) setup.
+
+           Enabling Server Side Encryption (SSE) also deploys a MinIO :minio-git:`KES <kes>` service in the Tenant to faciliate SSE operations.
+  
+           For more, see the `Github documentation <https://github.com/minio/kes/wiki>`__.
+
+         - :mc:`~kubectl minio tenant create --storage-class`
+           
+           Specify the storage class to use.
+
+           New MinIO tenants use the ``default`` storage class.
+           To specify a different storage class, add the :mc:`~kubectl minio tenant create --storage-class` flag.
+
+           The specified :mc-cmd:`~kubectl minio tenant create --storage-class` *must* match the ``storage-class`` of the Persistent Volumes (``PVs``) to which the ``PVCs`` should bind.
+
+           MinIO strongly recommends creating a Storage Class that corresponds to locally-attached volumes on the host machines on which the Tenant deploys. 
+           This ensures each pod can use locally-attached storage for maximum performance and throughput. 
+           See the :ref:`Deploy MinIO Tenant <minio-k8s-deploy-minio-tenant>` tutorial for guidance on creating Storage Classes for supporting the MinIO Tenant.
+
+      3) Generate a YAML file for further customizations
+
+         Other options exist for customizing a MinIO tenant.
+         To customize additional configurations, edit a YAML file.
+
+         Do a dry run of a tenant creation process and generate a YAML file with the command line options with the :mc:`~kubectl minio tenant create --output` flag.
+
+         When using this flag, the operator does **not** create the tenant.
+         Modify the generated YAML file as desired, then use ``kubectl apply -f <FILE>`` to manually create the MinIO tenant using the file.
+
+      4) Record the Access Information
+
+         When generating the tenant, the MinIO Operator displays the access credentials to use for the tenant.
+
+         .. important::
+   
+            This is the only time the credentials display.
+            They cannot be retrieved later.
+
```

Comment:
passive voice. Spitballing:

"Copy the displayed credentials to a secure location. MinIO does not display these credentials again."